src/services/player.js

```diff
@@ -0,0 +1,18 @@
+import { client } from './client';
+import { checkError } from './client';
+
+export async function fetchPlayer() {
+  const resp = await fetch(`${process.env.REACT_APP_SUPABASE_URL}/rest/v1/players?`, {
```

Comment:
Using the supabase HTTP URL totally works but for constancies sake we may want to use the supabase client for this function too